piplicenses.py

```diff
@@ -414,9 +414,22 @@ def get_sortby(args):
     return 'Name'
 
 
+def create_plain_vertical_output(args, output_fields):
```

Comment:
It's better to define a class that extends `PrettyTable`  than to define a new function. For example, `JsonPrettyTable`, `CSVPrettyTable`.

Reply:
I actually did play around with `PrettyTable` for some time trying to get it to match the [Angular output](https://angular.io/3rdpartylicenses.txt). The problem I kept running into is that fundamentally `PrettyTable` is trying to create a horizontal table, but in this mode we don't want any of that (no columns, padding, separators, etc).

I found myself basically trying to hack `PrettyTable` to undo the "table" part of it and it was looking really ugly. I realized that for this really simple vertical format that just manually looping was easier to read and reason about.

Do you know how to get `PrettyTable` to do this?

Reply:
For example, can't you define it like this and get the behavior you want?

```python
class AngularStyledTable(PrettyTable):

    def get_string(self, **kwargs):
        options = self._get_options(kwargs)
        rows = self._get_rows(options)

        output = ''
        for row in rows:
            for v in row:
                output += '{}\n'.format(v)

            output += '\n'

        return output
```

In my opinion, inheriting `PrettyTable` like other contributors works better with options like `--sort` and `--with-urls` and more.